src/blocker.rs

```diff
@@ -95,11 +100,28 @@ impl Blocker {
             }
         });
 
-        // If there is a match
+        // only match redirects if we have them set up
         let redirect: Option<String> = filter.as_ref().and_then(|f| {
-            if f.is_redirect() {
-                // TODO: build up redirect URL from matching resource
-                unimplemented!()
+            // If there is a match
+            if let Some(blocker_redirects) = self.resources.as_ref() {
+                // Filter redirect option is set
+                if let Some(redirect) = f.redirect.as_ref() {
+                    // And we have a matching redirect resource
+                    if let Some(resource) = blocker_redirects.get_resource(redirect) {
+                        let mut data_url: String;
+                        if resource.content_type.contains(';') {
+                            data_url = format!("data:{},{}", resource.content_type, resource.data);
+                        } else {
+                            data_url = format!("data:{};base64,{}", resource.content_type, base64::encode(&resource.data));
+                        }
+                        Some(data_url.trim().to_owned())
+                    } else {
+                        // TOOD: handle error - throw?
```

Comment:
Could this be logged somehow, even if its just mirroring the "didn't understand filter: X" stuff the current lib does?  Would be a nice, noisy reminder if there is some new filter format we don't support, something like that